src/components/segments/MeetupsContainer.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import _ from 'lodash';
+import moment from 'moment';
+
+import MeetupDate from './MeetupDate';
+
+class MeetupsContainer extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {
+      loading: true,
+      meetups: []
+    }
+  }
+
+
+  componentWillMount() {
+    const url = "https://codeforamerica.org/api/organizations/Code-for-Denver/upcoming_events?per_page=50";
+    const xmlhttp = new XMLHttpRequest();
+
+    xmlhttp.onreadystatechange = () => {
+      if (xmlhttp.readyState === 4 && xmlhttp.status === 200) {
+        var data = JSON.parse(xmlhttp.responseText);
+        var meetupData = data.objects;
+        _.forEach(meetupData, (value, key, collection) => {
```

Comment:
As of es5 (all modern browsers IE9+) arrays `forEach`, so you could also just have `meetupData.forEach(...`.

Actually, this could be a case for `map`, and you could chain it with lodash.

Eh never mind, it isn't as clean as I was imagining:
```js
const meetupDataSorted = _(data.objects)
  .map(value => { 
    value.start_time = moment(value.start_time); 
    return value 
  }).sortBy('start_time').value();
```